### PR TITLE
Add turkish localization

### PR DESCRIPTION
**Pull request by uAliFurkanY**

### CODE REVIEW

File: game-localization/localization_tr.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// GENERIC
+generic_by : tarafından
```
Comment (aduermael) [inline game-localization/localization_tr.txt:2]: Oh wow, is a lot of characters for "by". 😅
We may have to support multiline labels in buttons for that to fit... 
Screen Shot 2021-11-27 at 23 10 14

**Comment by uAliFurkanY** [comment]
Or phrase it differently. Does not fit with Turkish structure. The correct
way to say this would be "aduermael tarafından". By is reversed in Turkish.

**Comment by uAliFurkanY** [comment]
I'm not saying change the entire game for Turkish, but give us the ability
to choose if by comes first or last

**Comment by gdevillele** [comment]
> I'm not saying change the entire game for Turkish, but give us the ability to choose if by comes first or last

We have a way to allow this !
you can do 

```%s tarafından```

(%s is the placeholder for the author's name) 🙂

**Comment by uAliFurkanY** [comment]
ok done

**Review by gdevillele — APPROVED** [review]
Looks good to me 👍 
Thanks a lot !

**Pull request merged**